index.bs

```diff
@@ -235,6 +235,20 @@ An aggregatable trigger data is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Aggregatable dedup key</h3>
+
+An aggregatable dedup key is a [=struct=] with the following items:
+
+<dl dfn-for="aggregatable dedup key">
+: <dfn>dedup key</dfn>
+:: Null or a non-negative 64-bit integer.
```

Comment:
What is the benefit of making this nullable?

Reply:
We actually null check below to see whether a dedup key exists or not. It's set to null if it's not specified in the JSON or invalid format.

We need the dedup key to be nullable to allow developers to specify that there's no dedup key for a filter.